RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -144,6 +146,13 @@ void AliAnalysisTaskAO2Dconverter::UserCreateOutputObjects()
     Tracks->Branch("fTRDsignal", &fTRDsignal, "fTRDsignal/F");
     Tracks->Branch("fTOFsignal", &fTOFsignal, "fTOFsignal/F");
     Tracks->Branch("fLength", &fLength, "fLength/F");
+    if (fTaskMode) {
```

Comment:
We should explicitly check that this is equal to 1 (if we want to support more modes).
Moreover, I would then use an `enum` to specify the different task mode, just to have a cleaner code.

Reply:
I agree on this, It is now done!